lib/byebug/commands/continue_breakpoint.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "byebug/command"
+require "byebug/helpers/parse"
+
+module Byebug
+  #
+  # Allows the user to continue execution until the next breakpoint, if
+  # the breakpoint file or line be different, stop again
+  #
+  class ContinueBreakpointCommand < Command
+    include Helpers::ParseHelper
+
+    class << self
+      attr_writer :file_line, :file_path
+
+      def file_line
+        @file_line ||= 0
+      end
+
+      def file_path
+        @file_path ||= ""
+      end
+    end
+
+    def self.regexp
+      /^\s* c(?:ont(?:inue)?_)?(?:b(?:reak(?:point)?)?) \s*$/x
+    end
+
+    def self.description
+      <<-DESCRIPTION
+        c[ont[inue]_]b[reak[point]]
+        #{short_description}
+      DESCRIPTION
+    end
+
+    def self.short_description
+      "Runs until the same breakpoint, if breakpoint change stop again"
+    end
+
+    def keep_execution(file, line)
+      [self.class.file_path, self.class.file_line] == [file, line]
+    end
+
+    def reset_attributes
+      self.class.always_run = 0
+    end
+
+    def auto_run(frame)
+      return false unless self.class.always_run == 2
+
+      keep_execution(frame.file, frame.line) ? processor.proceed! : reset_attributes
+      true
+    end
+
+    def execute
+      return if auto_run frame
```

Comment:
The `frame` is available as an instance method, so I think we can avoid passing it around?

Reply:
Changed